packages/web-components/src/rating-display/rating-display.ts

```diff
@@ -0,0 +1,122 @@
+import { attr, FASTElement, nullableNumberConverter } from '@microsoft/fast-element';
+import { RatingDisplayColor, RatingDisplaySize } from './rating-display.options.js';
+
+/**
+ * The base class used for constructing a fluent-rating-display custom element
+ * @public
+ */
+export class RatingDisplay extends FASTElement {
+  /**
+   * The internal {@link https://developer.mozilla.org/docs/Web/API/ElementInternals | `ElementInternals`} instance for the component.
+   *
+   * @internal
+   */
+  public elementInternals: ElementInternals = this.attachInternals();
+
+  /**
+   * The color of the rating display icons.
+   *
+   * @public
+   * @default `marigold`
+   * @remarks
+   * HTML Attribute: `color`
+   */
+  @attr
+  public color: RatingDisplayColor = RatingDisplayColor.marigold;
+
+  /**
+   * Renders a single filled icon with a label next to it.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: `compact`
+   */
+  @attr({ attribute: 'compact', mode: 'boolean' })
```

Comment:
```suggestion
  @attr({ mode: 'boolean' })
```
Given the attribute and IDL attribute are the same, there's no reason to set it in the decorator